package.json

```diff
@@ -68,6 +71,11 @@
     "start": "python ./script/start.py",
     "test": "python ./script/test.py"
   },
+  "husky": {
+    "hooks": {
+      "precommit": "npm run check-clang-format \"npm run clang-format\""
```

Comment:
husky also supports`scripts.precommit` instead of `hustky.hooks.precommit`. But what you have here is more explicit and less magic. I don't feel strongly, but If we stick to this format, then the existing `prepack` and `prepush` scripts should probably be moved here too.